pkg/operator/v1alpha1helpers/helpers.go

```diff
@@ -207,3 +210,38 @@ func EnsureOperatorConfigExists(client dynamic.Interface, operatorConfigBytes []
 		}
 	}
 }
+
+type InstallConfigGetter func() (installertypes.InstallConfig, error)
```

Comment:
Do we really need this type abstraction?

Reply:
I kept it straight from what was in MCO, it seems like it just serves for [test stubbing](https://github.com/abhinavdahiya/machine-config-operator/blob/55872cc024432b18d687a8010d30456fe5ad9778/pkg/operator/render_test.go#L86). Was hoping to carry it here to try not and totally break their existing test but I'm fine with getting rid of it